src/java/org/apache/cassandra/db/SystemKeyspace.java

```diff
@@ -1862,17 +1865,61 @@ public static void resetPreparedStatements()
         preparedStatements.truncateBlockingWithoutSnapshot();
     }
 
-    public static int loadPreparedStatements(TriFunction<MD5Digest, String, String, Boolean> onLoaded)
+    public static int loadPreparedStatements(TriFunction<MD5Digest, String, String, Prepared> onLoaded)
```

Comment:
Hopefully it's ok to change the type signature here?  I don't expect this method is used outside of the codebase itself and tests.

Reply:
We need it to pull the size so it is justifiable imo